diff --git a/core/java/src/test/java/hara/truffle/StdFsTest.java b/core/java/src/test/java/hara/truffle/StdFsTest.java
--- a/core/java/src/test/java/hara/truffle/StdFsTest.java
+++ b/core/java/src/test/java/hara/truffle/StdFsTest.java
@@ -133,10 +133,7 @@ public class StdFsTest {
 
   private static Context newContext(Path root) {
     IOAccess io = IOAccess.newBuilder().fileSystem(new HaraMountedFileSystem(root)).build();
-    return Context.newBuilder(HaraLanguage.ID)
-        .currentWorkingDirectory(root)
-        .allowIO(io)
-        .build();
+    return Context.newBuilder(HaraLanguage.ID).allowIO(io).build();
   }
 
   private static void deleteTree(Path root) throws IOException {
